src/protocol/libp2p/bitswap/mod.rs

```diff
@@ -54,6 +54,9 @@ mod schema {
 /// Log target for the file.
 const LOG_TARGET: &str = "litep2p::ipfs::bitswap";
 
+/// Write timeout for outbound messages.
+const WRITE_TIMEOUT: Duration = Duration::from_secs(15);
```

Comment:
Is this the same constant libp2p uses?

Reply:
Should be in the same ballpark as substrate 15s timeout: https://github.com/paritytech/polkadot-sdk/blob/0404a8624964441011730e274c7a02972b63245c/substrate/client/network/src/bitswap/mod.rs#L111